index.html

```diff
@@ -15,22 +15,22 @@
                 </nav>
             </header>
             <div class = "container">
-                <h2>Hello! I am Eugene<br>
-                Glad to see you.
+                <h2>I'm Eugene.<br>
+                    Glad to see you!
                 </h2>
```

Comment:
- [ ] I would kindly suggest you make this an `h1` tag.
   - The main title (e.g., "I'm Marianne ...") is a `h1` tag.
      - If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.